includes/Feed/FeedProductFormatter.php

```diff
@@ -0,0 +1,236 @@
+<?php
+namespace SkyVerge\WooCommerce\Facebook\Feed;
+
+if ( ! defined( 'ABSPATH' ) ) {
+	exit;
+}
+
+use SkyVerge\WooCommerce\Facebook\Products;
+
+/**
+ * Class responsible for formatting product for feed file.
+ */
+class FeedProductFormatter {
+
+	const MAX_IMAGES_PER_PRODUCT = 5;
+
+	/**
+	 * Formats a product to be synced to Facebook.
+	 *
+	 * @see FeedDataExporter::attribute_variants for explanation why $attribute_variants are passed as reference.
+	 * @since 2.6.0
+	 * @param \WC_Facebook_Product $woo_product WooCommerce product object normalized by Facebook.
+	 * @param array                $attribute_variants Array of variants attributes.
+	 * @return array[string] product feed line data
+	 */
+	public function prepare_product_for_feed( $woo_product, &$attribute_variants ) {
```

Comment:
```suggestion
	public function prepare_product_for_feed( $fb_product, &$attribute_variants ) {
```

It seems like this param represents a `WC_Facebook_Product` product rather than a "Woo" one.